resources/metadata/gctsCreateRepository.yaml

```diff
@@ -0,0 +1,100 @@
+metadata:
+  name: gctsCreateRepository
+  description: Creates a Git repository on an ABAP system
+  longDescription: |
+    Creates a local Git repository on an ABAP system if it does not already exist.
+
+spec:
+  inputs:
+    secrets:
+      - name: credentialsId
```

Comment:
I think the credentials ID should be prefixed to not interfere with other credentials.

@OliverNocon are secret always evaluated with `GENERAL` scope.

Reply:
What naming would you suggest? "ABAPServerUsernamePasswordCredentialsId"?

Reply:
I noticed that we also have it like this in [abapEnvironmentPullGitRepo](https://github.com/SAP/jenkins-library/blob/master/resources/metadata/abapEnvironmentPullGitRepo.yaml#L15). So I guess we can also leave it like this.

Reply:
yes, `GENERAL` is okay.

We should apply the correct naming pattern though:
https://github.com/SAP/jenkins-library/blob/master/.github/CONTRIBUTING.md#jenkins-credential-handling